components/page-store/StoreDetail.tsx

```
@@ -0,0 +1,73 @@
+import Rating from 'react-rating';
+import { useTranslation } from 'react-i18next';
+
+import FollowStore from 'components/buttons/FollowStore';
+import Icon from 'components/Icon';
+import IStoreDetail from 'interfaces/StoreDetail';
+import styles from 'styles/page-store/StoreDetail.module.scss';
+
+interface Props {
+  store: IStoreDetail;
+  hasFollowed: boolean;
+  switchHasFollowed: () => void;
+}
+
+const RATING = 4;
+
+function StoreDetail({ store, hasFollowed, switchHasFollowed }: Props) {
+  const { t } = useTranslation();
+  const getContent = () => {
+    const ICON_INFO_ARR = [
+      {
+        name: 'facebook',
+        component: <Icon.Facebook />,
+      },
+      {
+        name: 'facebook',
+        component: <Icon.Twitter />,
+      },
+      {
+        name: 'pinterest',
+        component: <Icon.Pinterest />,
+      },
+      {
+        name: 'share',
+        component: <Icon.Share />,
+      },
+    ];
+    return ICON_INFO_ARR.map(({ name, component }) => (
+      <button type="button" key={name} className={styles['btn-block-share-file']}>{component}</button>
+    ));
+  };
+
+  return (
+    <div className={styles.main}>
+      <div>
+        <img src={store.banner_url ?? undefined} className={styles['row-banner']} />
```

We may need a default banner or hide this div if `banner_url` is `''` or `null`